runtime/long.go

```diff
@@ -487,6 +489,49 @@ func longRBinaryBoolOpSlot(fun func(x, y *big.Int) bool) *binaryOpSlot {
 	return &binaryOpSlot{f}
 }
 
+func longPow(f *Frame, v, w *Object) (*Object, *BaseException) {
+	var w_long *big.Int
+
+	v_long := toLongUnsafe(v).Value()
+	if w.isInstance(LongType) {
+		w_long = toLongUnsafe(w).Value()
+	} else if w.isInstance(IntType) {
+		w_long = big.NewInt(int64(toIntUnsafe(w).Value()))
+	} else {
+		return NotImplemented, nil
+	}
+
+	if w_long.Sign() < 0 {
+		// The result will be a float, so convert this and call the floating point function.
+		var v_float, w_float  *Object
+		var exception *BaseException
```

Comment:
Please use the name `raised` for raised exceptions since that's almost universal in the code